### ISSUE TITLE
Lead Dashboard: Dev

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#4294]
### Dependency
- #6229

### Overview
We need to create a dashboard so that it is easy for Dev Leads to know where to focus their effort to maintain a healthy backlog of issues for each complexity type while addressing time sensitive requirements.

### Action Items
- [ ] Create a dashboard below in a comment by using template
- [ ] Iterate
- [ ] Once stable, discuss with Leads best way to make it automatic

### Template
```
# Full report
Dev Dashboard 
Date:

Good First Issue front end & back end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257) :  
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 
[ERs](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-19403960): 

Small backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257):
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217):
ERs: 

Small front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257):
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-15235217):
ERs: 

Medium backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+medium%22#column-7198257): 
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217):
ERs: 

Medium front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257):  
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-15235217):
ERs: 

Large backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+++label%3A%22complexity%3A+large%22#column-7198257): 
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+large%22#column-15235217):
ERs: 

Large front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-7198257):
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-15235217): 
ERs: 
```

### Resources/Instructions
- [Project Board](https://github.com/hackforla/website/projects/7)
- [Emoji site with numbers](https://emojiterra.com/search/keycap/)

**Comment by ExperimentsInHonesty** [comment]
Good Morning!  Here is the Dev Dashboard  for today 2023-03-24
This time with items that need attention only, with the full report in the thread.  Trying different layouts to see what we like.  Feedback welcome.

Good Second Issue front end & back end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+good+second+issue%22#column-7198257): 1
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+good+second+issue%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 1 (time sensitive)
ERs: 0

Small front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257):  1
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-15235217) 5 (1 is time sensitive and has been sitting for a few days)
ERs: 1 (for tech lead)

Medium backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+medium%22#column-7198257):   1 (1 which is for dev leads) I moved the rest to new issue approval because they are half worked and need to be reviewed by a tech lead and cleaned up)
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217) 4 (3 of these need cleanup)
ERs: 3

Medium front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257): 0
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-15235217) 2
ERs: 2  (ready for dev lead)

Large front end
[Prioritized Backlog:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-7198257) 0
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-15235217) 2 with the [ready for dev lead label](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22+label%3A%22ready+for+dev+lead%22#column-15235217)
not much you can do here, except check the in progress, which I just did.
There was one person who was assigned by Bonnie, who never put their ETA on.  I have written Krista, and we will remove her from the issue on Sunday if no response. https://github.com/hackforla/website/issues/4136
Will check on this person on Sunday.  They estimated ETA on 2023-03-23 EOD and I dont' see a PR, etc. https://github.com/hackforla/website/issues/3821

**Comment by ExperimentsInHonesty** [comment]
# Full report
Dev Dashboard 2023-03-24

Good First Issue front end & back end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257) :  8 (7 are front end and backend; 1 is backend only)
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 0
[ERs](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-19403960): 0

Good Second Issue front end & back end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+good+second+issue%22#column-7198257): 1
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+good+second+issue%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 1 (time sensitive)
ERs: 0

Small backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257):  2
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217) 2 (1 is waiting on stakeholder, so not ready for dev lead; both are also front end)
ERs: 0

Small front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257):  1
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-15235217) 5 (1 is time sensitive and has been sitting for a few days)
ERs: 1 (for tech lead)

Medium backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+medium%22#column-7198257):   1 (1 which is for dev leads) I moved the rest to new issue approval because they are half worked and need to be reviewed by a tech lead and cleaned up)
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217) 4 (3 of these need cleanup)
ERs: 3

Medium front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257): 0
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-15235217) 2
ERs: 2  (ready for dev lead)

Large backend
[Prioritized Backlog:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+++label%3A%22complexity%3A+large%22#column-7198257) 6
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+large%22#column-15235217) 3
ERs: 6

Large front end
[Prioritized Backlog:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-7198257) 0
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-15235217) 2 with the [ready for dev lead label](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22+label%3A%22ready+for+dev+lead%22#column-15235217)
ERs: 0

**Comment by ExperimentsInHonesty** [comment]
# Full report
Dev Dashboard 2023-03-26

Good First Issue front end & back end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257) :  6 all are front end and backend)
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 0
[ERs](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-19403960): 0

Good Second Issue front end & back end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+good+second+issue%22#column-7198257): 1 (1 is time sensitive)
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+good+second+issue%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 1 (time sensitive)
ERs: 0

Small backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257):  4 (2 time sensitive and one is also front end)
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217) 1 (time sensitive and is both back and front end)
ERs: 0

Small front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257):  3 (1 is also backend time sensitive)
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-15235217) 3 (1 is time sensitive, two will be talked about in the Monday meeting and 1 #4269)
ERs: 2

Medium backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+medium%22#column-7198257):  3 (1 which is for dev leads) 
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217) 2 
ERs: 4

Medium front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257): 2 (1 time sensitive)
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-15235217) 2 (1 is ready for product)
ERs: 4  (ready for dev lead; one is also front end)

Large backend
[Prioritized Backlog:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+++label%3A%22complexity%3A+large%22#column-7198257) 5
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+large%22#column-15235217) 3
ERs: 5

Large front end
[Prioritized Backlog:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-7198257) 1
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-15235217) 2 with the [ready for dev lead label](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22+label%3A%22ready+for+dev+lead%22#column-15235217)
ERs: 0

**Comment by ExperimentsInHonesty** [comment]
# Summary of Priorities
Dev Dashboard 2023-03-26

1️⃣  **The new front end team members that have finished their good first issue and are ready to start on their second issue will be stalled after the second issue. Because we have just enough for the people already waiting below and there are potentially 9 people who are going to finish their second issue in the next week**

Small front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257): 3 (1 is also backend)
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-15235217) 3 
ERs: 2

Small backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257): 4 (2 time sensitive and one is also front end)
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217) 1 (time sensitive and is both back and front end)
ERs: 0

```
People who need or will need Small issues
- Maria Herzer, @mjh-projects, #3911 (could also do a backend)
- Karen G. Cecena, @karengcecena #4072 (could also do backend) I took an issue away from her because we was not available for 5 days and it was a time sensitive issue - check in on her on 03-29
- Janice Chan, @janice87, #4173 (frontend only, has a pr submitted)
- Drake Nguyen, @drakenguyen4000, #4174 (frontend only has a pr submitted)
- Nicolette Mabeza, @nicolettedmabeza, #3968 (frontend only)

```

---

2️⃣ **We have 9 people who need good second issues and only a total of 8 issues between good first and good second.  And we have an onboarding scheduled for Tuesday.  So we probably need at least a total of 30 good first and second issues (15 good first issues would be a good start)**

Good First Issue front end & back end
- [Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257) : 7 
- [New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 0
- [ERs](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-19403960): 0

Good Second Issue front end & back end
- [Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+good+second+issue%22#column-7198257): 1 
- [New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+good+second+issue%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 1 (time sensitive)
- ERs: 0
```
People who should take up a Good Second Issue next.  
They have all been messaged in their preworks (2023-03-26) to pickup a good first issue if there are no good second issues).
- Aine Fitzgerald, @annierm18, #3709
- Ellen Kellybrew, @EllenKellyb, #3777
- Alejandro Gomez , @agosmou #3787
- Jeristella, @jeristella, #3908
- Phuong Vu, @phuongv8, #3917 
- Victoria Jackson, @neonPurple, #3972
- Luis Sotelo, @LOSjr4 , #4070
- Justin Hang, @hang-justin, #4161
- Devlin Booth, @adevlinb, 4171
```

---

3️⃣  **We are just a little bit short of issues in prioritized backlog for the people queuing** 

Medium front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257): 2 
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-15235217) 2 (1 is ready for product)
ERs: 4 (ready for dev lead)
```
People waiting for front Medium Front End Issues
- Joshua Juarez, @str-xjua24, #3605 (frontend only)
- Kyra Kwak, @kyrakwak, #3825 (front end only)
- Joy Lu, @jlu9d2, #4073 (front end only)
- Ronald Paek, @ronaldpaek, #4074 (front end only) - off for a week

Also in the list below, because they can do either
- Adaugo Akaluso, @Adaakal, #3782 (frontend or backend)
- Angelina Hayrapetyan, @angieyan, #3813 (frontend or backend)
- Matthew Malane, @mattmalane, #4067 (frontend or backend)
- Jeff Lueck, @jefflueck, #4090 (frontend or backend)

```

--- 

4️⃣ **We are just a little bit short of issues in prioritized backlog for the people queuing**

Medium backend
There are X people on our team waiting for Medium issues 
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+medium%22#column-7198257): 3 (1 of which is for dev leads) 
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217) 2
ERs: 4
```
People waiting for Medium backend issue
- Adaugo Akaluso, @Adaakal, #3782 (frontend or backend)
- Angelina Hayrapetyan, @angieyan, #3813 (frontend or backend)
- Matthew Malane, @mattmalane, #4067 (frontend or backend)
- Jeff Lueck, @jefflueck, #4090 (frontend or backend)

````

--- 

5️⃣ **We are likely to need more Large frontend issues soon based on the 4 people doing medium issues now who are front end only.**

Large front end
[Prioritized Backlog:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-7198257) 1
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-15235217) 2 with the [ready for dev lead label](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22+label%3A%22ready+for+dev+lead%22#column-15235217)
ERs: 0

Large backend
[Prioritized Backlog:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+++label%3A%22complexity%3A+large%22#column-7198257) 5
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+large%22#column-15235217) 3
ERs: 5

```
People ready for Large issues (or about to be)
Kailey Kotynski, @kotynskm, #3778 (frontend or backend)
Cheuk Kay Mary Ng, @maryjng, #3788 (frontend or backend)
Marco Calderon, @mademarc, #3780 (front end only, is working on a medium now)
```

**Comment by ExperimentsInHonesty** [comment]
# Issues to close if no response

## Issues to close 30-30
- https://github.com/hackforla/website/issues/3829
- https://github.com/hackforla/website/issues/3830
- https://github.com/hackforla/website/issues/3910
- https://github.com/hackforla/website/issues/3919

## Issues to close 04-02
- https://github.com/hackforla/website/issues/3981
- https://github.com/hackforla/website/issues/4060
- https://github.com/hackforla/website/issues/4061
- https://github.com/hackforla/website/issues/4160
- https://github.com/hackforla/website/issues/4162

Responded, check on them in a few days
- https://github.com/hackforla/website/issues/4164
- https://github.com/hackforla/website/issues/4075 - JD will check in on him

At message them with the following
```
Since we have not heard from you.  I am going to go ahead and close this issue.  If you want to return to the website team in the future, feel free to reopen this issue and put it in the in progress column.
````

**Comment by roslynwythe** [comment]
- @ExperimentsInHonesty I closed all the issues under "Issues to close 3-30" and "Issue to close 04-02"  except for #4061 and  #4162 since they provided updates, and #4164 and #4075 sound like they're on track.

**Comment by ExperimentsInHonesty** [comment]
# Full report
Dev Dashboard 2023-04-06

Good First Issue front end & back end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257) :  20 (19 all also backend)
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 0
[ERs](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-19403960): 0

Good Second Issue front end & back end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+good+second+issue%22#column-7198257): 0
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+good+second+issue%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 1 
ERs: 0

Small backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257):  2
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217) 3 (1 is both back and front end)
ERs: 1 (for product)

Small front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257):  7
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-15235217) 2 (1 is also front back end)
ERs: 3 (2 for product)

Medium backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+medium%22#column-7198257):  2 (1 which is for dev leads) 
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217) 4
ERs: 1

Medium front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257): 2 
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-15235217) 1 
ERs: 3

Large backend
[Prioritized Backlog:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+++label%3A%22complexity%3A+large%22#column-7198257) 4
[New Issue Approval:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+large%22#column-15235217) 3, but (2 are for dev leads only)
ERs: 6

Large front end
[Prioritized Backlog:](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-7198257) 2
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-15235217) 4 
ERs: 1

**Comment by ExperimentsInHonesty** [comment]
# Full report
Dev Dashboard 2023-05-14

Good First Issue front end & back end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257) :  21
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 
[ERs](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-19403960): 

Small backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257):1
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 5
ERs: 1

Small front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257):
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-15235217):
ERs: 

Medium backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+medium%22#column-7198257): 
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217):
ERs: 

Medium front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257):  
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-15235217):
ERs: 

Large backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+++label%3A%22complexity%3A+large%22#column-7198257): 
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+large%22#column-15235217):
ERs: 

Large front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-7198257):
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-15235217): 
ERs:

**Comment by anjolaaoluwa** [comment]
Good First Issue front end & back end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257) : 19
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 4
[ERs](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-19403960):0

Small backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257):1
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217): 2
ERs: 1

Small front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257):5
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-15235217):2
ERs:

Medium backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+medium%22#column-7198257):1
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+back+end%2Fdevops%22#column-15235217):2
ERs:

Medium front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257):1
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-15235217):3
ERs:

Large backend
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+++label%3A%22complexity%3A+large%22#column-7198257):3
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+large%22#column-15235217):4
ERs:

Large front end
[Prioritized Backlog](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-7198257):0
[New Issue Approval](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+large%22+label%3A%22role%3A+front+end%22#column-15235217):3
ERs:

**Comment by roslynwythe** [comment]
Hi @anjolaaoluwa please leave an update on this issue.  Thank you.

**Comment by bonniewolfe** [comment]
Why did we close the issue?

**Comment by bonniewolfe** [comment]
slide: https://docs.google.com/presentation/d/1hZCNMZewl4L_so1ceGMOuP0mTbaBkUuwzAbBPgcGYfU
Dashboard: https://lookerstudio.google.com/u/0/reporting/f42df1c1-99df-4fa1-b719-7e135168bfa3/page/e1JSB

**Comment by roslynwythe** [comment]
@ExperimentsInHonesty This is fantastic work and it will be very valuable to dev leads, but I noticed that when I select "front end only", the Prioritized Backlog column does not appear.  I checked the Project Board and in fact there are no "front end only" issues in the Prioritized Backlog column, but it might be better if the Prioritized Backlog column was displayed even if the number of issues is zero.  
<details><summary>Screenshot -  "front end only" when no front end only issues exist in Prioritized Backlog.  That column is not displayed at all</summary>

![image](https://github.com/hackforla/website/assets/5314153/9bd558b8-453b-4cc8-8457-c29d1006390d)

</details>

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe please walk us through that when we see you later today.

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe can you check this when the dashboard is working again. https://github.com/hackforla/website/issues/6229
